src/app/file-manager/reducers/downloadables.reducer.ts

```diff
@@ -61,6 +61,21 @@ export function DownloadablesReducer(state = initialState, action: Action): Down
                 })
             };
 
+        case DownloadablesActions.TOGGLE_ITEM_SELECTION:
+
+            const tmp_tog = state.items.filter(item => {
```

Comment:
this seems like an odd way to modify a list of items, normally I'd use `map()` for this, e.g.
```
const tmp_tog = state.items.map(item => item.selected = !item.selected);
```
Although this modifies `state.items` in place so there's no need to declare `tmp_tog`, using `state.items` should have the same effect.